README.md

```diff
@@ -80,15 +83,27 @@ This table contains an overview of all created and tracked AIPs. It should be up
 |:--|:--|
 | `Draft` | Drafts are currently in process and not ready for review. No corresponding GH Issue will be created.|
 | `In Review` | AIPs are ready for community review and feedback. See suggestions on providing feedback below. |
+| `Ready for Review` | AIPs are ready for Gatekeeper review and feedback. |
 | `Accepted `| AIPs has been accepted and will be implemented soon. |
 | `Rejected` | A community decision has been made to not move forward with an AIP at this time.| 
 | `On Hold` | Some information is missing or prerequisites have not yet been completed. | 
 
 ## Layer
 * Framework
 * Core (Blockchain)
+* Gas
+* Cryptography
+* Interface
+* Platform
```

Comment:
@gregnazario , @banool can you pick one? I don't know if we need both of these?

Reply:
Let's go with Platform